expense-tracker/src/db/procedures/fetch_wallet_details_v1dot0.sql

```diff
@@ -0,0 +1,10 @@
+DELIMITER $$
+create procedure fetch_wallet_details_v1dot0 (
+    IN in_wallet_id bigint
+)
+BEGIN
+    select * from t_wallet_details where f_id = in_wallet_id;
+    select f_id, f_first_name from t_user_details where f_email IN (select f_email_id from t_user_wallet_ref where f_wallet_id = in_wallet_id);
```

Comment:
My point being 
- if there's a screen in an app (lets say whatsapp profile. do they show your groups in your proile ? nope.)  where we show user details sending wallets to that response doesn't make sense. In this case we will have to write a different API to get List<Wallet> for a given user id.
- Similarly in case of a wallet just wallet detials isn't sufficient where its necessary to show the people involved in it. SO we need a ref to User object from wallet.